pkg/actors/actors.go

```diff
@@ -546,6 +547,27 @@ func (a *actorsRuntime) Call(ctx context.Context, req *internalv1pb.InternalInvo
 	}
 
 	if err != nil {
+		if errors.Is(err, ErrReminderCanceled) {
```

Comment:
Reminder cancellation is not intended to be valid in a normal actor call context. It seems a little strange that we'd include this logic here.

Reply:
This function is called from the `workflow_actor.go` file. This func makes a call to `a.isActorLocal` which then does a check if its an internal actor. I thought the reminder cancel error was relevant to internal actors. I can add to the conditional to check if its an internal actor only then go into this logic. Or are you saying this isn't related to reminders in general?

Reply:
I'll rm this. I think you are saying this is not needed here